mllib/src/main/scala/org/apache/spark/ml/evaluation/MultilabelClassificationEvaluator.scala

```diff
@@ -34,12 +34,13 @@ import org.apache.spark.sql.types._
  */
 @Since("3.0.0")
 @Experimental
-class MultilabelClassificationEvaluator (override val uid: String)
+class MultilabelClassificationEvaluator @Since("3.0.0") (@Since("3.0.0") override val uid: String)
```

Comment:
It's OK if it's for consistency, but if the whole class is "since 3.0" then implicitly so is everything in it. Do we really do this in a lot of the code? it just seems especially noisy to mark the constructor and then all its args this way

Reply:
Almost all  of the ml classes do this way. I am doing this simply for consistency. I am OK to remove.

Reply:
OK leave it in

Reply:
I also do not feel strongly about it.